translations/hu.json

```diff
@@ -0,0 +1,2984 @@
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds add-guild` paranccsal."
+      ],
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik."
+      ],
+      "rankNotFound": [
+        "A **{rank}** rang nem létezik a **{guild}** nevű guild-ban."
+      ],
+      "rankNotAdded": [
+        "A **{guild}** nevű guild **{rank}** rangja nincs regisztrálva a bottal.\n",
+        "Regisztráld a `/config guilds add-rank` paranccsal."
+      ],
+      "addedRequirement": [
+        "A **{requirement}** követelmény hozzá lett adva a **{guild}** nevű guild **{rank}** rangjához!"
+      ]
+    }
+  },
+  "config guilds remove-rank-requirement": {
+    "name": "rang-követelmény-eltávolítás",
+    "description": "Eltávolít egy követelményt egy Hypixel guild rangról.",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A guild amelyiknél a rang van."
+      },
+      {
+        "name": "rank",
+        "description": "A rang, ahonnan eltávolítod a követelményt."
+      },
+      {
+        "name": "követelmény",
+        "description": "Az eltávolítandó követelmény.",
+        "choices": [
+          "Hypixel szint",
+          "Képesség átlag",
+          "Slayer tapasztalat",
+          "Minion helyek",
+          "Catacombs szint",
+          "Fairy Souls",
+          "Súlyozás",
+          "API beállítások bekapcsolva"
+        ]
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds add-guild` paranccsal."
+      ],
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik."
+      ],
+      "removedRequirement": [
+        "A **{requirement}** követelmény el lett távolítva a **{guild}** nevű guild **{rank}** rangjáról!"
+      ]
+    }
+  },
+  "config guilds edit-rank-requirement": {
+    "name": "rang-követelmény-módosítás",
+    "description": "Módosítja egy Hypixel guild rangjának követelményét.",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A guild amelyiknél a rang van."
+      },
+      {
+        "name": "rank",
+        "description": "A rang, amelyiknek módosítod a rangját."
+      },
+      {
+        "name": "követelmény",
+        "description": "A módosítandó követelmény",
+        "choices": [
+          "Hypixel szint",
+          "Képesség szint",
+          "Slayer tapasztalat",
+          "Minion helyek",
+          "Catacombs szint",
+          "Fairy Souls",
+          "Súlyozás"
+        ]
+      },
+      {
+        "name": "érték",
+        "description": "Az érték, amire a követelményt állítod."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincsen ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds add-guild` parancshoz."
```

Comment:
The whole command name should be translated, including top level
Should also be in backticks for `this effect`